kolibri/plugins/facility/assets/src/views/DataPage/SyncInterface/index.vue

```diff
@@ -65,7 +67,7 @@
                     <CoreMenuOption
                       :style="{ 'cursor': 'pointer', textAlign: 'left' }"
                       :label="$tr('register')"
-                      @select="displayModal(Modals.REGISTER_FACILITY)"
+                      @select="handleRegister(false)"
```

Comment:
**_super_ non-blocking, completely just curious**: i was wondering, _if_ `handleRegister()` is always called with `false` here, if it might be cleaner to have this local `handleRegister()` function have a default value of `displaySkipOption = false`, or something similar, rather than a hardcoded `false` here, unless that would interact with some intended functionality

Reply:
After taking a look at the code, I was able to find a cleaner way to handle changing `displaySkipOption`.

Using  `this.displaySkipOption = Boolean(displaySkipOption)` within the `handleRegister` function and calling `handleRegister()` on line 70 with no argument instead of `false` has the same outcome with cleaner code. Thanks for pointing that out!

Reply:
nice!!